### DISCUSSION TITLE
Add Empty Dir

### DISCUSSION THREAD

**Discussion by russellmann** [maennchen/ZipStream-PHP#191] [Q&A] [answered]
Hello,

I have a circumstance where I may want to add an empty directory to the zip. How would one go about that?

**Comment by russellmann** [comment, marked as answer]
Found. Sorry.

https://github.com/maennchen/ZipStream-PHP/issues/78

**Reply by ahmed-bhs** [reply]
$zip->addFile('directory/', ''); does not work with 2.1.0 version
